chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# This does what is necessary directly after upgraded node is rebooted
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+set -x
+
+<% if @use_ha %>
+
+echo "Unsetting the pre-upgrade node attribute..."
+
+crm node attribute $(hostname) set pre-upgrade false
```

Comment:
So currently I'm executing it from `models/api/upgrade` ... still may be not the right place and time...